device_automation/device_automation.html

```diff
@@ -4,17 +4,17 @@
         <select id="node-input-server"></select>
     </div>
     <div class="form-row">
-        <label for="node-input-name"> 实体名称</label>
-        <input type="text" id="node-input-name" placeholder="实体名称">
+        <label for="node-input-name"> Entity name</label>
+        <input type="text" id="node-input-name" placeholder="Entity name">
     </div>
     <div class="form-row">
-        <label for="node-input-action"> 操作</label>
-        <input type="text" id="node-input-action" placeholder="点击、双击、三连击、长按">
+        <label for="node-input-action"> operate</label>
+        <input type="text" id="node-input-action" placeholder="Click, double click, three courses, long press">
```

Comment:
three courses ?? I think you mean triple click

Reply:
This is google translate. The work is INCOMPLETE. why did you create a PR??

Reply:
wow this is funny to read